Yoast/wordpress-seo
# Remove facebook insights

## Summary

This PR can be summarized in the following changelog entry:

* Remove all Facebook Insights functionality as it's no longer supported.

## Test instructions

This PR can be tested by following these steps:

* Go to Social → Facebook and see that it's gone.

## Quality assurance


Fixes #8261
